migrations/20201021191942_fs_type_field.sql

```diff
@@ -0,0 +1,4 @@
+DROP TYPE IF EXISTS fs_type;
```

Comment:
This migration is not the newest one anymore, is that going to cause issues?

Reply:
It's fine in this case because we haven't shipped any of the other ones to production yet.